actions/create_delivery_project_in_supr.yaml

```diff
@@ -6,7 +6,7 @@ enabled: true
 entry_point: supr.py
 parameters:
     timeout:
-        default: 10
+        default: 600
```

Comment:
Is SUPR really so slow that it can finish within 10 seconds?

Reply:
It has happened, but maybe it is more probable that it, in those cases, is not responding at all. Maybe it's a better idea to make this task retry if it fails?

Reply:
It depends. We're just doing a simple HTTP POST without any re-try logic at https://github.com/arteria-project/arteria-packs/blob/master/actions/supr.py#L80. And according to http://docs.python-requests.org/en/master/user/advanced/#timeouts this will hang forever, as long as the network hasn't failed completely, in which case a `ConnectionError` should be raised immediately (http://docs.python-requests.org/en/master/user/quickstart/#errors-and-exceptions). 

I.e. the `requests.post()` call can be waiting indefinitely for the syscall `connect()` to return, for initiating the connection; or it can have an established connection, but be waiting for more data to be received. 

In the case of a `ConnectionError` it is obviously more relevant to retry later. In the other cases... well, I'm a bit surprised that it can take them more than 10 seconds to be honest, if it isn't anything on our end (Stackstorm slow for example, which causes the action to take longer than 10 seconds to spawn - but the HTTP POST to SUPR might be quick). 

I guess there is no harm in increasing the timeout here drastically though, it just doesn't feel pretty. :)